### PR TITLE
Check for errors in the returned body

### PR DESCRIPTION
**Pull request by jenslind**
https://github.com/remixz/publish-release/issues/3

### CODE REVIEW

**Comment by jenslind** [comment]
ping @remixz :)

File: index.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ PublishRelease.prototype.publish = function publish () {
           'User-Agent': 'publish-release ' + pkg.version + ' (https://github.com/remixz/publish-release)'
         }
       }, function (err, res, body) {
-        if (err) return callback(err)
+        if (err || body.errors) return callback(err || body.errors)
```
Comment (marcbachmann) [inline index.js]: It's probably better to return error instances. `body.errors`, is that a string or an object?

Comment (maxcnunes) [inline index.js]: I think is better pass ahead the whole error object. As you can see in this [message](https://github.com/remixz/publish-release/pull/10#issue-118297225) the `erros` property does not have the whole information about the error.

**Comment by Aluxian** [comment]
This is handled by this line, now: https://github.com/remixz/publish-release/blob/master/index.js#L105

Right? Please let me know if I'm wrong.

**Pull request closed without merging**